scripts/publish.js

```diff
@@ -118,6 +118,14 @@ function copyScssFiles() {
   return fs.copy(`${coreLibDir}/scss`, `${distTarget}/scss`, { filter: onlyScssFiles });
 }
 
+function copyScssUtilsFile() {
+  console.log('Copying SCSS utils file...');
+  const onlyScssFiles = (input) => ['', '.scss'].includes(path.extname(input));
+  return fs.copy(`${angularLibDir}/scss/_utils.scss`, `${distTarget}/scss/_utils.scss`, {
+    filter: onlyScssFiles,
+  });
+}
+
```

Comment:
Perhaps plop in a comment stating this is temporary and link to relevant issue when it is created :D